Events/CacheResolveEvent.php

```diff
@@ -12,8 +12,22 @@
 namespace Liip\ImagineBundle\Events;
 
 use Symfony\Component\EventDispatcher\Event;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\EventDispatcher\Event as ContractsEvent;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface as ContractsEventDispatcherInterface;
 
-class CacheResolveEvent extends Event
+if (is_subclass_of(EventDispatcherInterface::class, ContractsEventDispatcherInterface::class)) {
```

Comment:
I'm using such test because in previous build I got the 4.2 event dispatcher installed alongside contracts 1.1. In such configuration, `Symfony\Component\EventDispatcher\EventDispatcherInterface` does not implements `Symfony\Contracts\EventDispatcher\EventDispatcherInterface` and so does require our event to extend `Symfony\Component\EventDispatcher\Event`.

Reply:
yep, that makes sense to me :+1: